docs/docs/cmd/util/accesstoken/accesstoken-get.mdx

```diff
@@ -30,6 +30,12 @@ The `util accesstoken get` command returns an access token for the specified res
 
 If the URL of your SharePoint tenant has been previously retrieved, you can use `sharepoint` as the resource to retrieve access token for SharePoint. To verify if the URL of your SharePoint tenant has been previously retrieved, use the [`spo get`](../../spo/spo-get.mdx) command. To explicitly set the URL of your SharePoint tenant use the [`spo set`](../../spo/spo-set.mdx) command.
```

Comment:
Since this is covered in the new admonition, I think we can remove this one.

Reply:
Removed.